packages/perseus-core/src/widgets/radio/radio-util.test.ts

```diff
@@ -0,0 +1,88 @@
+import {getRadioPublicWidgetOptions} from "@khanacademy/perseus-core";
+
+import type {PerseusRadioWidgetOptions} from "../../data-schema";
+
+describe("getRadioPublicWidgetOptions", () => {
+    it("should return the correct public options without any answer data", () => {
+        // Arrange
+        const options: PerseusRadioWidgetOptions = {
+            choices: [
+                {
+                    content: "$-8$ and $8$",
+                    correct: false,
+                    clue: "The square root operation ($\\sqrt{\\phantom{x}}$)...",
+                    widgets: {},
+                },
+                {
+                    content: "$-8$",
+                    correct: false,
+                    clue: "While $(-8)^2=64$, the square root operation...",
+                    widgets: {},
+                },
+                {
+                    content: "There is no such input value.",
+                    isNoneOfTheAbove: true,
+                    correct: true,
+                    clue: "This is a none of the above option.",
+                    widgets: {
+                        "sorter 1": {
+                            type: "sorter",
+                            options: {
+                                correct: ["test"],
+                                padding: true,
+                                layout: "horizontal",
+                            },
+                        },
+                    },
+                },
+            ],
+            hasNoneOfTheAbove: false,
+            countChoices: false,
+            randomize: false,
+            multipleSelect: false,
+            deselectEnabled: false,
+            onePerLine: false,
+            displayCount: null,
+            noneOfTheAbove: false,
+        };
+
+        // Act
+        const publicWidgetOptions = getRadioPublicWidgetOptions(options);
+
+        // Assert
+        expect(publicWidgetOptions).toEqual({
+            choices: [
+                {
+                    content: "$-8$ and $8$",
+                    widgets: {},
+                },
+                {
+                    content: "$-8$",
+                    widgets: {},
+                },
+                {
+                    content: "There is no such input value.",
+                    isNoneOfTheAbove: true,
+                    widgets: {
+                        "sorter 1": {
+                            type: "sorter",
+                            options: {
+                                correct: ["test"],
+                                padding: true,
+                                layout: "horizontal",
+                            },
+                        },
+                    },
```

Comment:
Seems like this could potentially be unnecessary, especially since we don't really use this functionality, but I wanted to make sure it preserves it since we're specifically keeping `widgets` as part of the choice public data.